pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1585,18 +1589,35 @@ func (r *ReconcileInfinispan) serviceExternal(m *infinispanv1.Infinispan) *corev
 			Namespace: m.ObjectMeta.Namespace,
 			Labels:    lsService,
 		},
-		Spec: corev1.ServiceSpec{
+	}
+
+	if externalServiceType == "LoadBalancer" {
```

Comment:
It would be nice to use constants from the typesv1 for better supporting and validation

Reply:
yeah sure, thanks!